### PR TITLE
Change plugin command to use Select Menu

### PR DESCRIPTION
**Pull request by jkcoxson**
Looks pretty sick

### CODE REVIEW

File: coreCommands.js [hunk 1]
```diff
@@ -114,56 +168,32 @@ module.exports = class plugin extends EventEmitter {
...
-                    embeds: [embed],
-                    components: [comp]
-                });
-            });
 
 
         }
 
     }
+
```
Comment (sb64) [inline coreCommands.js:196]:
```suggestion
```

File: coreCommands.js [hunk 2]
```diff
@@ -178,4 +208,17 @@ function generateColor() {
         toSend += allowedHex[Math.floor(Math.random() * allowedHex.length)];
     }
     return (toSend);
-}
\ No newline at end of file
+}
+function getPluginMenu() {
```
Comment (sb64) [inline coreCommands.js:212]:
```suggestion

function getPluginMenu() {
```

File: package.json [hunk 3]
```diff
@@ -13,13 +13,17 @@
     "author": "jkcoxson",
     "license": "None",
     "dependencies": {
+        "@discordjs/voice": "^0.5.3",
         "discord.js": "github:discordjs/discord.js",
+        "ffmpeg": "^0.0.4",
+        "ffmpeg-static": "^4.4.0",
         "node-fetch": "^2.6.1",
+        "sodium": "^3.0.2",
         "winston": "^3.3.3"
```
Comment (sb64) [inline package.json:22]: There's an issue with our dependencies, we'll probably want to fix that in a separate PR before we merge this one

**Review by sb64 — APPROVED** [review]
Looks good to me, just 2 nits

File: package.json (hunk 3 above)
Comment (sb64) [reply package.json:22]: https://github.com/CamelBot/CamelBot/runs/3050694719

Comment (jkcoxson) [reply package.json:22]: I canceled it on purpose because it just ran, but if you look it ran the first time successfully

**Pull request merged**